__tests__/app.test.js

```diff
@@ -26,6 +26,23 @@ describe('/api/topics', () => {
     })
 })
 
+describe('/api', () => {
+    test('GET 200 - responds with a object describing all the endpoints on the API', () => {
+        return request(app)
+        .get('/api')
+        .expect(200)
+        .then(({body}) => {
+            const { endpoints } = body
+            expect(typeof endpoints).toBe('object')
+            expect(Array.isArray(endpoints)).toBe(false)
+            expect(endpoints).toMatchObject({
+                "GET /api": { "description": expect.any(String) },
+                "GET /api/topics": { "description": expect.any(String), "queries": [], "exampleResponse": { "topics": [{ "slug": expect.any(String), "description": expect.any(String) }] }},
+            })
```

Comment:
If you require in the endpoints file you can then expect that the file will always look like whatever is in it at the time. As you'll be adding to the file as you go along with all of your endpoints you want the test to be dynamic.